### ISSUE TITLE
Some confusion about Post-Pre-Training

### ISSUE THREAD

**Issue by shaoxiang** [jiachengliu3/OpenTrajBooster#1]
Thanks for the great work on open source. I noticed that in the paper, **Post-Pre-Training** is described as a key innovation, but the code seems use the same fine-tune script as GR00T. Is Post-Pre-Training essentially a fine-tuning phase with redirected data, or are there additional technical differences (e.g., loss functions, training strategies) that distinguish it from standard fine-tuning?
Also, is the **l2aggle/PPTmodel4UnitreeG1** model trained using commands similar to the following?
```
dataset_list=(
  "l2aggle/Agibot2UnitreeG1Retarget"
)

python scripts/gr00t_finetune.py \
  --base-model-path ../models/GR00T-N1.5-3B \
  --dataset-path ${dataset_list[@]} \
  --num-gpus 1 \
  --output-dir ./save/multiobj_pick_WBC/ \
  --data-config openwbc_g1 \
  --embodiment-tag new_embodiment \
  --batch-size 128 \
  --max_steps 20000 \
  --report-to tensorboard
```
